src/test/java/org/apache/sling/resourceresolver/impl/mapping/MapEntriesTest.java

```diff
@@ -2074,4 +2102,28 @@ public void run() {
             }
         }
     }
+
+    @Test
+    public void testIsValidAliasPath() throws Exception {
```

Comment:
Have you considered making that method have `package` level access by removing the `private` modifier? It might make the code simpler with no ill side-effects.